resources/default_pipeline_environment.yml

```diff
@@ -337,13 +338,17 @@ steps:
     logSuccessfulMavenTransfers: false
   mtaBuild:
     buildTarget: 'NEO'
-    mtaBuildTool: classic
+#    mtaBuildTool: classic
+    mtaBuildTool: 'cloudMbt'
```

Comment:
incompatible change.

@CCFenner when was this switch planned?

Reply:
Not yet decided if we should/could do this at all. Tool availability will cease in May but Docker images will persist, so it could be still used.